pkg/kine/server/server.go

```diff
@@ -12,19 +13,26 @@ import (
 	healthpb "google.golang.org/grpc/health/grpc_health_v1"
 )
 
+// The emulated etcd version is returned on a call to the status endpoint. The version 3.5.13, indicates support for the watch progress notifications.
+// See: https://github.com/kubernetes/kubernetes/blob/beb696c2c9467dbc44cbaf35c5a4a3daf0321db3/staging/src/k8s.io/apiserver/pkg/storage/feature/feature_support_checker.go#L157
+const emulatedEtcdVersion = "3.5.13"
```

Comment:
Re-opening the discussion on the emulatedEtcdVersion flag here @berkayoz, @bschimke95 and @marco6.

It would not make sense for users to drop the version, we may only bump it in the future if some new fancy feature requires it.

If a user wishes to disable features they should do so in the api-server.
As a consequence, we do not have to make this emulatedEtcdVersion configurable.